app/src/main/java/org/jellyfin/androidtv/model/itemtypes/LiftedItems.kt

```diff
@@ -36,4 +39,14 @@ class Episode(original: BaseItemDto) : PlayableItem(original) {
 	}
 }
 
-class Movie(original: BaseItemDto) : PlayableItem(original)
+class Person(original: BaseItemPerson) {
```

Comment:
Can be moved to a seperate file as it doesn't use our base class (yet?)

Reply:
Oh, forgot to ask that. I wonder if persons should remain separate or become an item as well. Currently they are their entirely own thing in the apiclient